bug: error in `make lint`

## Description

```bash
# triplez @ TripleZs-MacBook-Pro in ~/Code/envd on git:main o [23:33:27] 
$ make lint                                                                                                  
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /bin
golangci/golangci-lint info checking GitHub for latest tag
golangci/golangci-lint info found version: 1.46.2 for v1.46.2/darwin/amd64
install: /bin//golangci-lint: Read-only file system
make: *** [/Users/triplez/go/bin/golangci-lint] Error 71

# triplez @ TripleZs-MacBook-Pro in ~/Code/envd on git:main o [23:33:35] C:2
$ go env GOPATH
/Users/triplez/go
```

## Reproduction

Run `make lint` on a macOS machine with normal user privilege (`/bin` directory is protected by super user).

## Additional Info

Use double `$` if you want to enter `$` for a shell command in the makefile.

--- a/Makefile
+++ b/Makefile
@@ -102,7 +102,7 @@ lint: $(GOLANGCI_LINT)
 	@$(GOLANGCI_LINT) run
 
 $(GOLANGCI_LINT):
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $$(go env GOPATH)/bin
 
 mockgen-install:
 	go install github.com/golang/mock/mockgen@v1.6.0

